python/pyspark/sql/context.py

```diff
@@ -311,6 +312,24 @@ def registerJavaFunction(
         )
         return self.sparkSession.udf.registerJavaFunction(name, javaClassName, returnType)
 
+    # TODO(andrew): delete this once we refactor things to take in SparkSession
+    def _inferSchema(self, rdd: RDD, samplingRatio: Optional[float] = None) -> StructType:
```

Comment:
What is this addition?

Reply:
Sorry, I think I made a mistake during the rebase.

Reply:
Thanks @dongjoon-hyun, the code in this file should be okay now.